### PR TITLE
update options for less-loader@6.0.0 expectation

### PR DESCRIPTION
**Pull request by aitsvet**
This is to update options for less-loader@6.0.0 expectation:

https://github.com/ant-design/ant-design/issues/23624#issuecomment-625381410

Done all coverage and audit commands.

### CODE REVIEW

File: lib/craco-antd.js [hunk 1]
```diff
@@ -51,14 +51,19 @@ const overrideWebpackConfig = ({ context, webpackConfig, pluginOptions }) => {
...
-  lessLoaderOptions.modifyVars = modifyVars;
-  // javascriptEnabled: true is suggested in the Ant Design docs:
-  // https://ant.design/docs/react/customize-theme#Customize-in-webpack
-  lessLoaderOptions.javascriptEnabled = true;
+  lessLoaderOptions.lessOptions = {
+    modifyVars: modifyVars,
+    // javascriptEnabled: true is suggested in the Ant Design docs:
+    // https://ant.design/docs/react/customize-theme#Customize-in-webpack
+    javascriptEnabled: true
+  };
```
Comment (daniel-hauser) [inline lib/craco-antd.js:66]: This is dangerous since it overrides `lessOptions` and passes only `modifyVars` and `javascriptEnabled`

Comment (daniel-hauser) [reply lib/craco-antd.js:66]: @ndbroadbent

Comment (ndbroadbent) [reply lib/craco-antd.js:66]: Ah, good point. I will revert this commit now and wait for a PR that solves this in a better way.

Comment (aitsvet) [reply lib/craco-antd.js:66]: Thanks for the review!
https://github.com/DocSpring/craco-antd/pull/38

**Pull request merged**